borderless/explainer.md

```diff
@@ -0,0 +1,512 @@
+- [additional-windowing-controls explainer](https://github.com/ivansandrk/additional-windowing-controls/blob/main/awc-explainer.md)
+- [ChromeOS app-details](crbug.com/1225871) (Googlers-only)
+- [go/cros-privacy-indicators-design](go/cros-privacy-indicators-design)
+  (Googlers-only)
+
+## Table of contents
+
+- [Borderless Explainer](#borderless-explainer)
+  - [Table of contents](#table-of-contents)
+  - [Introduction](#introduction)
+  - [Example of a complex title bar with window-controls-overlay](#example-of-a-complex-title-bar-with-window-controls-overlay)
+  - [Goals](#goals)
+  - [Non-goals](#non-goals)
+  - [Proposal](#proposal)
+    - [1. Setting the `display_override` to borderless](#1-setting-the-display_override-to-borderless)
+    - [2. Enabling draggable regions](#2-enabling-draggable-regions)
+    - [3. CSS display-mode media query for borderless](#3-css-display-mode-media-query-for-borderless)
+    - [4. Leveraging Window Management permission for borderless](#4-leveraging-window-management-permission-for-borderless)
+      - [Enabling Window Management](#enabling-window-management)
+      - [Disabling Window Management](#disabling-window-management)
+    - [5. Isolated Web App check](#5-isolated-web-app-check)
+  - [Related features](#related-features)
+    - [Displaying app’s origin in the app settings UI](#displaying-apps-origin-in-the-app-settings-ui)
+    - [Privacy indicators](#privacy-indicators)
+  - [Other notable details](#other-notable-details)
+    - [“Settings and more” three-dot button](#settings-and-more-three-dot-button)
+  - [Demo](#demo)
+    - [manifest.json](#manifestjson)
+    - [index.html](#indexhtml)
+    - [style.css](#stylecss)
+  - [Considered Alternatives](#considered-alternatives)
+    - [Having a separate permission for borderless](#having-a-separate-permission-for-borderless)
+    - [WCO extension](#wco-extension)
+      - [Reasonings for discarding](#reasonings-for-discarding)
+    - [New JS APIs](#new-js-apis)
+      - [Reasonings for discarding](#reasonings-for-discarding-1)
+  - [Security Considerations](#security-considerations)
+    - [Spoofing risks](#spoofing-risks)
+    - [Out-of-scope Navigation](#out-of-scope-navigation)
+    - [Opening a popup from a borderless PWA](#opening-a-popup-from-a-borderless-pwa)
+    - [Iframes](#iframes)
+
+## Introduction
+
+[VDI](https://www.softwaretestinghelp.com/best-vdi-software/#What_Is_Virtual_Desktop_Infrastructure)
+(Virtual Desktop Infrastructure) providers attempt to mirror remote applications
+to the local desktop. For example, the VDI provider's cloud is running a Text
+Processor (a desktop application window), which is to be presented on the client
+desktop as if the application was running locally.
+
+Installed web apps define their display mode in the manifest.json using display
+and
+[display_override](https://github.com/WICG/display-override/blob/main/explainer.md)
+fields. Currently
```

Comment:
I would not enumerate all the values we currently support, since this might become out of date.